### PR TITLE
Add .Net Standard-related content to documentation

### PR DESCRIPTION
**Pull request by RobGallo**
#### Describe the change

Relevant after the creation of the SystemAbstractions .Net Standard project and the SystemAbstractionsTests .Net Core project

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: docs/NewProject.md [hunk 1]
```diff
@@ -2,43 +2,60 @@
...
-### For all projects
-1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
-2. Right-click on the project and select Properties.
+### All project types
+
+1. Add your new project to the Axe.Windows Solution (`src\AxeWindows.sln`).
+2. Before creating a pull request, verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
+3. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
+
+### For all .Net Framework projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
### For all .NET Framework projects
```

File: docs/NewProject.md [hunk 2]
```diff
@@ -2,43 +2,60 @@
...
 2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
-   - Currently .NET Framework 4.7.1 is the target. 
+   - Currently .NET Framework 4.7.1
 3. In the build tab, set the following for both Debug and Release configurations:
    1. "Warning level" to 4.
    2. "Treat warnings as errors" to "All".
 
-### For *non-test* projects only
-1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:
+#### For *production (not test)* .Net Framework projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
#### For *production (not test)* .NET Framework projects
```

File: docs/NewProject.md [hunk 3]
```diff
@@ -2,43 +2,60 @@
...
+   1. Remove the following lines, as well as the commented lines above them: <br>
       ```
       [assembly: AssemblyVersion("1.0.*")]
       [assembly: AssemblyFileVersion("1.0.0.0")]
       ```
-4. Verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
-5. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
 
-### For *test* projects only
+#### For *test* .Net Framework projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
#### For *test* .NET Framework projects
```

File: docs/NewProject.md [hunk 4]
```diff
@@ -2,43 +2,60 @@
...
+
 1. Close the solution and use your text editor to add the following line to your `.csproj` file to properly configure [Strong Naming](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/strong-named-assemblies) for your assembly:<br>
    `<Import Project="..\..\build\delaysign.targets" />`
-2. Verify that Visual Studio can successfully load and build the entire solution in both Debug and Release. Ensure that the tests are appropriately discovered in the test explorer, and that they all run successfully.
+2. Build the entire solution in both Debug and Release. Ensure that the tests are appropriately discovered in the test explorer, and that they all run successfully.
+
+### .Net Standard and .Net Core projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
### .NET Standard and .NET Core projects
```

File: docs/NewProject.md [hunk 5]
```diff
@@ -2,43 +2,60 @@
...
+
+#### For *production (not test)* .Net Standard/Core projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
#### For *production (not test)* .NET Standard/Core projects
```

File: docs/NewProject.md [hunk 6]
```diff
@@ -2,43 +2,60 @@
...
+
+In a text editor, add the following line:<br>
+`<Import Project="..\..\build\NetStandardRelease.targets" />`
+
+#### For *test* .Net Standard/Core projects
```
Comment (DaveTryon) [inline docs/NewProject.md]: Capitalize .NET
```suggestion
#### For *test* .NET Standard/Core projects
```

File: docs/NewProject.md (hunk 4 above)
Comment (DaveTryon) [inline docs/NewProject.md]: I totally understand the .NET Standard work, but do we have any current plans to add .NET Core projects? I'm not sure that we can realistically document them if we're not using them to validate the documentation.

File: docs/NewProject.md [hunk 7]
```diff
@@ -2,43 +2,60 @@
...
+1. Add your new project to the Axe.Windows Solution (`src\AxeWindows.sln`).
+2. Before creating a pull request, verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
+3. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
+
+### For all .Net Framework projects
+
+1. Right-click on the project and select Properties.
 2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
-   - Currently .NET Framework 4.7.1 is the target. 
+   - Currently .NET Framework 4.7.1
```
Comment (DaveTryon) [inline docs/NewProject.md]: >Currently .NET Framework 4.7.1 [](start = 5, length = 30)

I get the intent of this line, but the phrasing feels like "pigeon English"

File: docs/NewProject.md [hunk 8]
```diff
@@ -2,43 +2,60 @@
...
-Do the following when adding a new project:
 
-### For all projects
-1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
-2. Right-click on the project and select Properties.
+### All project types
+
+1. Add your new project to the Axe.Windows Solution (`src\AxeWindows.sln`).
+2. Before creating a pull request, verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
+3. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
```
Comment (DaveTryon) [inline docs/NewProject.md]: "alongside" is one word
```suggestion
3. If your project requires NuGet dependencies which need to be installed alongside it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
```

File: docs/NewProject.md [hunk 9]
```diff
@@ -2,43 +2,60 @@
...
    ```
 2. Close the solution and use your text editor to make the following changes to your `.csproj` file to properly configure the version and signing options:
-   1. Add the following line with the other `.cs` files):<br>
+   1. Add the following line along side the existing `<Compile>` statements:<br>
    `<Compile Include="$(TEMP)\AxeWindowsVersionInfo.cs" />`
    2. Add the following below the last ItemGroup:<br>
    ```
    <ItemGroup>
-      <DropSignedFile Include="$(OutDir)\[your project name].dll" />
+      <DropSignedFile Include="$(TargetPath)" />
```
Comment (DaveTryon) [inline docs/NewProject.md]: >DropSignedFile [](start = 7, length = 14)

Do we still need DropSignedFile? I thought it was for digital signing only.

**Comment by DaveTryon** [comment]
> CI | Builds the NuGet package that will be referenced by code that uses the library.

Out of scope for this PR, but the "CI" project name was intended for a very different scenario than the NuGet distribution that we now use. We might do well to rename this to something a more relevant to our current strategies. Maybe "NuGet" or "NuGetPackage" or something along those lines...

---
Refers to: docs/solution.md:40 in 5843856. [](commit_id = 584385654cbfe84b8914be73fdf77f6054c41258, deletion_comment = False)

File: docs/NewProject.md (hunk 7 above)
Comment (RobGallo) [reply docs/NewProject.md]: It's not actually pigeon English; it's just punctuated incorrectly. I kept the original formatting initially, but I'll change it.

File: docs/NewProject.md (hunk 4 above)
Comment (RobGallo) [reply docs/NewProject.md]: The SystemAbstractionsTests project is .NET Core. That was the alternative option to .NET Framework when creating a test project, although I don't know why one couldn't make the test project .NET Standard.

File: docs/NewProject.md (hunk 7 above)
Comment (RobGallo) [reply docs/NewProject.md]: Mostly it's the punctuation that makes this awkward. I'll fix it.

File: docs/NewProject.md (hunk 9 above)
Comment (RobGallo) [reply docs/NewProject.md]: The only way I know to find out is to remove it, create a PR, and create a signed build. I don't object, but I think it's out of scope for this PR.

File: docs/NewProject.md (hunk 4 above)
Comment (DaveTryon) [reply docs/NewProject.md]: Fair enough

---
In reply to: [305451883](https://github.com/microsoft/axe-windows/pull/117#discussion_r305451883) [](ancestors = 305451883)

File: docs/NewProject.md [hunk 10]
```diff
@@ -2,43 +2,59 @@
...
+#### For *production (not test)* .NET Framework projects
+
+1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:<br>
    ```
    Microsoft.VisualStudioEng.MicroBuild.Core
    Microsoft.CodeAnalysis.FxCopAnalyzers
    ```
 2. Close the solution and use your text editor to make the following changes to your `.csproj` file to properly configure the version and signing options:
-   1. Add the following line with the other `.cs` files):<br>
+   1. Add the following line along side the existing `<Compile>` statements:<br>
```
Comment (DaveTryon) [inline docs/NewProject.md]: "alongside" is one word
```suggestion
   1. Add the following line alongside the existing `<Compile>` statements:<br>
```

**Pull request merged**